test/e2e/bindata/assets/02_operandserviceaccount.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: openshift-descheduler-operand
```

Comment:
You should not need to deploy the operand SA and CR. The operator is responsible for rendering those:
- https://github.com/openshift/cluster-kube-descheduler-operator/blob/master/bindata/assets/kube-descheduler/operandclusterrole.yaml
- https://github.com/openshift/cluster-kube-descheduler-operator/blob/master/bindata/assets/kube-descheduler/operandserviceaccount.yaml
- https://github.com/openshift/cluster-kube-descheduler-operator/blob/master/bindata/assets/kube-descheduler/operandclusterrolebinding.yaml

Reply:
Ok